docs/getting_started.rst

```diff
@@ -351,6 +351,9 @@ The cog guides are formatted the same. They're divided into 3 sections:
 
         Arguments enclosed in ``[ ]`` means that the argument is **optional**
         for the command; you can decide to use it or not.
+        
+        If your argument includes spaces like ``Hello world!`` you need to place
```

Comment:
Hmmmm this is not true for every command though... so generelizing here can leave to confusion.

Reply:
I'm not aware of any differantiation. How could the end user tell they need or don't need quotes?